src/panic.md

```diff
@@ -0,0 +1,98 @@
+# Panic
+
+Rust provides the ability to "panic" upon encountering an error that is not
+part of a function's signature; such an error is typically not expected to be
```

Comment:
This could use some rewording.  What's not part of the function's signature, the ability to panic, or the error encountered?

Reply:
The error: "an error that is not part of a function's signature."

It's been so long since I wrote this that I had completely forgotten writing this explanation of what a "panic" is (and I was surprised to learn that we don't already have a section in the reference dedicated to this). I think my intent here was to formalize [the way the book explains it](https://doc.rust-lang.org/stable/book/ch09-01-unrecoverable-errors-with-panic.html): "Sometimes bad things happen in your code, and there’s nothing you can do about it."

Do you have any concrete suggestions? I suppose I could rearrange what I have somewhat:

> Rust provides a mechanism to prevent a function from returning normally, and instead "panic," which is a response to an error condition that is typically not expected to be recoverable within the context in which the error is encountered.

I don't know whether that's clearer.

Reply:
Applied.